git_machete/docs.py

```diff
@@ -380,9 +380,9 @@
         <b>`sync`:</b>
 
             Synchronizes with the remote repository:
-                * deletes untracked managed branches with no downstream branch,
                 * deletes unmanaged branches and also checks out open PRs for the current user associated with the Github token,
```

Comment:
Pls also update the docs for `clean` to make the ordering of operations clear there